docs/source/api/aihwkit.simulator.rst

```diff
@@ -23,4 +22,4 @@ Submodules
 .. toctree::
    :maxdepth: 4
 
-   aihwkit.inference
```

Comment:
`aihwkit.simulator.noise_models` is legacy any should be removed. `aihwkit.inference` is the correct sub module

Reply:
removed.